tests/spec/005-status-types.js

```diff
@@ -18,20 +18,9 @@ test('shows followers-only vs regular in home timeline', async t => {
     .expect($(`${getNthStatusSelector(3)} .status-toolbar button:nth-child(2)`).hasAttribute('disabled')).ok()
 })
 
-test('shows direct vs followers-only vs regular in notifications', async t => {
+test('hides status toolbar on notification page', async t => {
```

Comment:
I need to figure out getting these test running locally and change it so we have a test that shows the status bar is missing on all notifications apart from mentions...

Reply:
Your current test looks good to me. In case it helps, I use a Podman container running Ubuntu 20.04 that basically runs the steps that the CI uses to install everything: 

https://github.com/nolanlawson/pinafore/blob/4a6907bbdc72659a86065ebc62afcc45e8aa2de2/.github/workflows/e2e-tests-readonly.yml#L46-L56

Reply:
Thanks appreciated! Will try this tomorrow